### ISSUE TITLE
JSONSerializer does not push payload's `included` records into the store

### ISSUE THREAD

**Issue by tstirrat** [emberjs/data#3549]
I am using `DS.JSONSerializer` with `DS.EmbeddedRecordsMixin` while trying to get emberfire compatible with 1.13.x (firebase/emberfire#283).

The embedded records are being correctly extracted into the JSON-API `included` property, but they are not being pushed into the store. This results in the error: 

```
You looked up the 'posts' relationship on a 'author' with id author_1 but some of the associated records were not loaded.
```

Initial payload given to `serializer.normalize`:

```json
{
  "id": "author_1",
  "name": "Tim",
  "posts": [
    {
      "id": "post_1",
      "title": "Post 1"
    }
  ]
}
```

Serializer normalizes this into:

```json
{
  "data": {
    "id": "author_1",
    "type": "author",
    "attributes": {
      "name": "Tim"
    },
    "relationships": {
      "posts": {
        "data": [
          {
            "id": "post_1",
            "type": "post"
          }
        ]
      }
    }
  },
  "included": [
    {
      "id": "post_1",
      "type": "post",
      "attributes": {
        "title": "Post 1"
      },
      "relationships": {}
    }
  ]
}
```

Here is a minimal test to show you what is happening:

http://jsfiddle.net/3eff3bsz/

The culprit is [_normalizeResponse #443](https://github.com/emberjs/data/blob/v1.13.5/packages/ember-data/lib/serializers/json-serializer.js#L443) not passing through the `included` property.

I'm willing to make a PR for this but do I branch off the `release` branch to get it into 1.13.x?

**Comment by bmac** [comment]
Hi @tstirrat. Thanks for tracking down the issue. Go ahead and open a pr against the master branch and prefix your commit message with [BUGFIX release], then the Ember Data team will cherry-pick the commit onto the release branch.

**Comment by tstirrat** [comment]
ok, thanks, will do

**Comment by wecc** [comment]
Thanks for the issue @tstirrat!

This is indeed a bug and should be easily solved by take `included` into account [here](https://github.com/emberjs/data/blob/v1.13.5/packages/ember-data/lib/serializers/json-serializer.js#L453) and [here](https://github.com/emberjs/data/blob/v1.13.5/packages/ember-data/lib/serializers/json-serializer.js#L457).

**Comment by fivetanley** [comment]
closed via #3550
